openrtb_ext/imp_openweb.go

```diff
@@ -0,0 +1,9 @@
+package openrtb_ext
+
+// ExtImpOpenWeb defines the contract for bidrequest.imp[i].ext.openweb
+type ExtImpOpenWeb struct {
+	SourceId    int     `json:"aid"`
+	PlacementId int     `json:"placementId,omitempty"`
+	SiteId      int     `json:"siteId,omitempty"`
+	BidFloor    float64 `json:"bidFloor,omitempty"`
+}
```

Comment:
Nitpick: It's preferred in Go to use `ID` instead of `Id`:

```
type ExtImpOpenWeb struct {
	SourceID    int     `json:"aid"`
	PlacementID int     `json:"placementId,omitempty"`
	SiteID      int     `json:"siteId,omitempty"`
	BidFloor    float64 `json:"bidFloor,omitempty"`
}
```